packages/cli/src/commands/preview.ts

```diff
@@ -1,9 +1,17 @@
 import { previewExam } from '@digabi/exam-engine-rendering'
 import { Ora } from 'ora'
 
-export default async function preview({ exam, spinner }: { exam: string; spinner: Ora }): Promise<string> {
+export default async function preview({
+  exam,
+  port,
+  spinner,
+}: {
+  exam: string
+  port: number
```

Comment:
The argument is optional, so let's mark its type properly.

```suggestion
  port?: number
```

Reply:
Done.